docs/tutorials/laravel.md

```diff
@@ -106,6 +106,19 @@ lando php -v
 
 You can also run `lando` from inside your app directory for a complete list of commands.
 
+Environment Config
+-----
+
+By default, Laravel comes with a `.env` configuration file set to use `homestead`. You will want to modify the following `.env` key value pairs:
+
+```
+APP_URL=http://myproject.lndo.site
+DB_HOST=database
+DB_DATABASE=laravel
+DB_USERNAME=laravel
+DB_PASSWORD=laravel
+```
+
```

Comment:
@jcandan can you add a line here that indicates if the app is already started they need to restart the app in order for lando to pick up on new or updated entries to the `.env` file?

Maybe something like:

If your app is already running be sure to restart it to have the changes picked up.